plugins/zns/zns.c

```diff
@@ -571,6 +620,54 @@ static int set_zone_desc(int argc, char **argv, struct command *cmd, struct plug
 	return nvme_status_to_errno(err, false);
 }
 
+
+static int flush_zone(int argc, char **argv, struct command *cmd, struct plugin *plugin)
+{
+	const char *desc = "Flush Explicit ZRWA Range";
+	const char *zllba = "The last LBA of the zone to be flushed";
+	const char *timeout = "timeout value, in milliseconds";
+
+	int err, fd;
+
+	struct config {
+		__u64	zllba;
+		__u32	namespace_id;
+		__u32	timeout;
+	};
+
+	struct config cfg = {};
+
+	OPT_ARGS(opts) = {
+		OPT_UINT("namespace-id", 'n', &cfg.namespace_id,  namespace_id),
+		OPT_SUFFIX("last-lba",   'l', &cfg.zllba,         zllba),
```

Comment:
The description of this field is defined as: 

  This field specifies the LBA of the highest-numbered logical block that is requested to be flushed by this Zone Send Action command. 

in the specification. I would recommend to rename the field to "flush-to-lba" or similar. Similarly the field description should be updated similarly.